platforms/chibios/drivers/spi_master.h

```diff
@@ -78,6 +78,7 @@ extern "C" {
 void spi_init(void);
 
 bool spi_start(pin_t slavePin, bool lsbFirst, uint8_t mode, uint16_t divisor);
+bool spi_start_extended(pin_t slavePin, bool lsbFirst, uint8_t mode, uint16_t divisor, bool cs_active_low);
```

Comment:
From a future-proofing perspective, perhaps we should be looking at something more like this:
```suggestion

typedef struct spi_start_config_t {
	pin_t slavePin;
	bool lsbFirst;
	uint8_t mode;
	uint16_t divisor;
	bool cs_active_low;
} spi_start_config_t;

bool spi_start(pin_t slavePin, bool lsbFirst, uint8_t mode, uint16_t divisor);
bool spi_start_extended(spi_start_config_t *config);
```

Reply:
Implemented in 3d206c57530dbe3b363b6c3faa2c88e4bead8d2b